docs/README.adoc

```diff
@@ -34,14 +34,44 @@ link:./integrating-libaktualizr.adoc[integrating-libaktualizr.adoc] - How to use
 
 Additional documentation intended for developers that may need to use the libaktualizr API should refer to link:https://advancedtelematic.github.io/aktualizr/index.html[the doxygen output on github].
 
+[NOTE]
+====
+The link above is for the doxygen docs on master. Doxygen docs for the following tagged versions are also available:
+
+* https://advancedtelematic.github.io/aktualizr/2018.8/index.html[2018.8]
+* https://advancedtelematic.github.io/aktualizr/2018.9/index.html[2018.9]
+* https://advancedtelematic.github.io/aktualizr/2018.11/index.html[2018.11]
+* https://advancedtelematic.github.io/aktualizr/2018.12/index.html[2018.12]
+* https://advancedtelematic.github.io/aktualizr/2018.13/index.html[2018.13]
+====
+
 === Updating doxygen on github
 
-To update the doxygen documentation on github, you will need to do something like the following:
+To update the doxygen documentation for master on github, you will need to do something like the following:
 
-1. In an aktualizr repo, run `make docs` in the build directory.
+1. In an aktualizr repo, run `make doxygen` in the build directory.
 1. Clone a second aktualizr repo and run `git checkout gh-pages`.
-1. In the second repo, run `git rm -r ./*`.
+1. In the second repo, run `rm ./* search/* `.
```

Comment:
What's the preference for this version of `rm`?

Reply:
Doxygen only creates one directory, search. Everything else is in the root. We no longer want to wipe the repo every time, because then you'd have to regenerate each of the tags every time you updated master.

Reply:
Right you are. Now I understand. Thanks!

Also, I technically didn't approve this yet. I would have as soon as I read the answer here, though, so there's no problem. But just sayin'!

Reply:
Oops! I don't know how I missed that, sorry.